src/GraphQL.Tests/Types/ComplexGraphTypeTests.cs

```diff
@@ -33,6 +33,7 @@ internal class TestObject
         [Description("Super secret")]
         public string? someString { get; set; }
         [Obsolete("Use someInt")]
+        [DefaultValue(true)]
```

Comment:
Ooops
>  System.InvalidOperationException : The required input field 'someBoolean' of an Input Object 'TestObject' has not default value so `@deprecated` directive must not be applied to this input field. To deprecate an input field, it must first be made optional by either changing the type to nullable or adding a default value.

And here is an example of breaking changes that I talked about. Not tooling, just "bad-written" schema that was valid before and is not valid now.